internal/bitwarden/bw/client.go

```diff
@@ -110,17 +136,44 @@ func (c *client) EditObject(obj Object) (*Object, error) {
 }
 
 func (c *client) GetObject(obj Object) (*Object, error) {
-	out, err := c.cmdWithSession("get", string(obj.Object), obj.ID).RunCaptureOutput()
-	if err != nil {
-		if string(out) == "Not found." {
-			return nil, ErrNotFound
+	var out []byte
+	var err error
+	if string(obj.Object) == "attachment" {
+		var tmpObj Object
+		out, err = c.cmdWithSession("get", "item", obj.ItemId).RunCaptureOutput()
+		if err != nil {
+			if string(out) == "Not found." {
+				return nil, ErrNotFound
+			}
+			return nil, err
 		}
-		return nil, err
-	}
 
-	err = json.Unmarshal(out, &obj)
-	if err != nil {
-		return nil, unmarshallError("get object", err, out)
+		err = json.Unmarshal(out, &tmpObj)
+		if err != nil {
+			return nil, err
+		}
+		result := filterAttachments(tmpObj.Attachments, func(val Attachment) bool {
```

Comment:
This is a bit more fragile than it needs to be. Given that we already know the attachment's ID and item ID from the creation, we can retrieve the attachment in one call without having to search for the attachment by name. It's also faster (one `bw` command less)